app/controllers/support_interface/providers_controller.rb

```diff
@@ -12,5 +12,16 @@ def sync
       SyncFromFind.perform_async
       redirect_to action: 'index'
     end
+
+    def update_all_courses
+      @provider = Provider.find(params[:provider_id])
+
+      if @provider.courses.update_all(open_on_apply: true)
+        flash[:success] = "Successfully updated all courses"
+        redirect_to support_interface_provider_path(@provider)
+      else
```

Comment:
Will you ever get here? I think `update_all` returns the number of rows affected not boolean.

Reply:
Removed the if statement